src/ape/api/compiler.py

```diff
@@ -35,6 +36,19 @@ def get_versions(self, all_paths: List[Path]) -> Set[str]:
             Set[str]: A set of available compiler versions.
         """
 
+    @abstractmethod
```

Comment:
Instead of making this abstract, we can have it return {} by default? This way, it won't be a breaking change.
We can later on decide to change it to abstract if it makes sense (such as pre-0.5)

Reply:
can you let me know if I did this right in the newest commit?